packages/yourloops/lib/units/utils.tsx

```diff
@@ -25,7 +25,7 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-export const MMOLL_PER_MGDL = 0.0555;
+export const MGDL_PER_MMOL = 18;
```

Comment:
As described here (it should be the same rule in all DBL products):
https://docs.google.com/spreadsheets/d/1P7b41oHoV73t5OgXFe22vo1Wso38u9ixgk3v0O81nKY/edit#gid=620459482

```suggestion
export const MGDL_PER_MMOL = 18.01577;
```